logging/include/Logging.hpp

```diff
@@ -111,9 +111,9 @@ class Logger {
 
 }  // namespace logging
 
-#define LOG_COMMON(logger, level, s)                              \
-  if ((logger).getLogLevel() <= level) {                          \
-    (logger).print(level, __PRETTY_FUNCTION__) << s << std::endl; \
+#define LOG_COMMON(logger, level, s)                                                                              \
+  if ((logger).getLogLevel() <= level) {                                                                          \
+    (logger).print(level, __PRETTY_FUNCTION__) << s << " | [SQ:" << std::to_string(getSeq()) << "]" << std::endl; \
```

Comment:
It breaks log4cplus formatting configuration, we shouldn't do it hardcoded.

Reply:
This is not what this "sequence number" is about, it is a thread local counter which allows checking if log messages are missing.